test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -10,6 +11,7 @@
     collectMetrics: true
   workloads:
   - name: 500Nodes
+    labels: [fast]
```

Comment:
Do you depend on this label? If not, prefer to remove it since it has no practical use.

Reply:
I have used it when running benchmarks manually to measure the effect of my changes in https://github.com/kubernetes/kubernetes/pull/115796. The 5000 node workloads are a bit too slow for that and aren't needed (the "fast" tests already run long enough to see the effect of changes).